scipy/sparse/_compressed.py

```diff
@@ -1407,6 +1407,48 @@ def _divide_sparse(self, other):
             return out
 
 
+    def broadcast_to(self, shape):
+        if self.format != 'csr':
+            raise NotImplementedError('broadcasting not implemented for'
+                                      'CSC format.')
+        
+        old_shape = self.shape
+        
+        if old_shape == shape:
+            return self.copy()
+        
+        if len(shape) != 2:
+            raise ValueError("Target shape must be a tuple of length 2.")
+
+        # Ensure the old shape can be broadcast to the new shape
+        if any((o != 1 and o != n) for o, n in zip(old_shape, shape)):
```

Comment:
Can we use `np.broadcast_shapes(old_shape, shape)` here?

Reply:
Changed at https://github.com/scipy/scipy/pull/21412/commits/9cdbce7203f0e4bd3420ff27054a46dde5669c90.